meta-phosphor/common/recipes-kernel/linux/linux-obmc.bb

```diff
@@ -0,0 +1,26 @@
+DESCRIPTION = "Linux kernel for OpenBMC"
+SECTION = "kernel"
+LICENSE = "GPLv2"
+
+KBRANCH ?= "dev"
+
+inherit kernel
+require recipes-kernel/linux/linux-yocto.inc
+
+SRC_URI = "git://github.com/openbmc/linux;protocol=git;branch=${KBRANCH}"
+SRC_URI += "file://obmc-bsp.scc \
+            file://obmc-bsp.cfg \
+            file://obmc-bsp-user-config.cfg \
+            file://obmc-bsp-user-patches.scc \
+           "
+
+LINUX_VERSION ?= "4.3-rc1"
+LINUX_VERSION_EXTENSION ?= "-openbmc"
```

Comment:
We want to end up with the git sha in the kernel version string, so we know where it came from. I'm not familiar with these rules, what will we get with your current setup?